docs/source/collect/logs.md

```diff
@@ -62,6 +62,10 @@ For duration string format see [time.ParseDuration](https://pkg.go.dev/time#Pars
 ##### `limits.maxLines`
 The number of lines to include, starting from the newest.
 
+##### `Limits.maxBytes`
+The maximum file size of a collected pod log, is currently set to 5MB as this will ensure a support bundle will not grow excessively due
+to large logfiles. Neither will the logs contain information that is too old or no longer relevant.
+
```

Comment:
please include the format for this field - can I specify 10MB or do I need to write that in bytes as an integer?